src/Features/Core/Portable/ConvertToInterpolatedString/AbstractConvertPlaceholderToInterpolatedStringRefactoringProvider.cs

```diff
@@ -219,23 +251,79 @@ bool ContainsIndex(string stringLiteralText, string indexString)
         }
     }
 
-    private static async Task<Document> CreateInterpolatedStringAsync(
+    protected override async Task FixAllAsync(
         Document document,
-        TInvocationExpressionSyntax invocation,
-        TArgumentSyntax placeholderArgument,
-        IMethodSymbol invocationSymbol,
-        TInterpolatedStringExpressionSyntax interpolatedString,
-        bool shouldReplaceInvocation,
+        ImmutableArray<TextSpan> fixAllSpans,
+        SyntaxEditor editor,
+        string? equivalenceKey,
         CancellationToken cancellationToken)
     {
         var syntaxFacts = document.GetRequiredLanguageService<ISyntaxFactsService>();
+
+        var root = await document.GetRequiredSyntaxRootAsync(cancellationToken).ConfigureAwait(false);
+        var semanticModel = await document.GetRequiredSemanticModelAsync(cancellationToken).ConfigureAwait(false);
+
+        var normalizedSpans = new NormalizedTextSpanCollection(fixAllSpans);
+
+        using var _ = ArrayBuilder<SyntaxNode>.GetInstance(out var stack);
+        stack.Push(root);
+
+        while (stack.TryPop(out var node))
```

Comment:
new FixAll.  it walks down the tree (for the parts of it that intersect the passed in spans of interest).  it then examines any invocations it finds, replacing them if analysis passes.